### DISCUSSION TITLE
Innova Butler web server

### DISCUSSION THREAD

**Discussion by Marsondiego** [danielrivard/homeassistant-innova#186] [General]
Hi everyone,
has someone tried to extend for integrating in HA Innova Butler? (https://www.innovaenergie.com/en/products/home-automation/butler-pro/butler-pro-touch/ ). It's a central controller which control Airleaf fancoil (and heatpump) connected throuh RS485.
If needed some raw output from it, when querying something (starting FC, changing temp, programming calendar), I could export it.
Thank you

**Comment by egidiocatini** [comment]
follow

**Comment by fatmacheto** [comment]
Following as well. Innova fan coils are great but want to stay away from the wi-fi.

**Comment by Marsondiego** [comment]
Hi, since I'm totally unable to write a single line of code, ask some helps to Claude to create the integration... it's definetely very far from perfect, but somehow seems to work for integration with Butler Touch (and probably also with the Butler Pro). 
https://github.com/Marsondiego/butlertouch
Please be lenient and don't shoot the pianist! :-)
